plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisConnectorConfig.java

```diff
@@ -220,13 +221,27 @@ public RedisConnectorConfig setRedisPassword(String redisPassword)
         return this;
     }
 
+    public boolean isUseSsl()
+    {
+        return useSsl;
+    }
+
+    @Config("redis.use-ssl")
+    @ConfigSecuritySensitive
+    @ConfigDescription("Whether SSL is enabled or not. Default is no")
+    public RedisConnectorConfig setUseSsl(boolean useSsl)
+    {
+        this.useSsl = useSsl;
+        return this;
+    }
+
     public boolean isHideInternalColumns()
     {
         return hideInternalColumns;
     }
 
     @Config("redis.hide-internal-columns")
-    @ConfigDescription("Whether internal columns are shown in table metadata or not. Default is no")
+    @ConfigDescription("Whether internal columns are shown in table metadata or not. Default is yes")
```

Comment:
Revert unrelated change.

Reply:
Understood. Should I create a dedicated commit for that though? The `no` is incorrect and also at odds with the `.rst` generated documentation (which is correctly stating that the default is `true`).

Reply:
Unfortunately, this sentence is misleading. 

`hideInternalColumns = true` means that internal columns are **_not_** shown, so the existing **_no_** is correct. 
Also, bootstrap log prints the default config values. There's no need to mention the value in `@ConfigDescription`. If you still want to fix this line in this PR, yes please separate a commit.

Reply:
Ah of course :man_facepalming:, my bad! I think it was just me not being smart then and this can be left untouched. I already reverted my "no to yes" change.